ruby_event_store/lib/ruby_event_store/subscriptions.rb

```diff
@@ -33,6 +33,42 @@ def all_for(event_type)
     private
     attr_reader :local, :global, :thread
 
+    class Store
+      def initialize
+        @subscriptions = Hash.new {|hsh, key| hsh[key] = [] }
```

Comment:
If we don't intend to store same subscription multiple times, I'd rather see `Set` here